Sprint-2/implement/vat.js

```diff
@@ -8,3 +8,10 @@
 // Given a number,
 // When I call this function with a number
 // it returns the new price with VAT added on
+
+
+function priceWithVAT(productPrice) {
```

Comment:
Oh we have a pound sign as well. What do we think - could this cause problems in a larger program?

Reply:
Thanks to mention that. I corrected it.